src/directives/ClickToAccept.tsx

```diff
@@ -178,7 +196,12 @@ const ClickToAccept = ({ directive, status, mutate }: DirectiveProps) => {
                 <FormControl error={showErrors} sx={{ mb: 3, mx: 0 }}>
                     <FormControlLabel
                         control={
-                            <Checkbox value={acknowledgedDocuments} required />
+                            <Checkbox
+                                disabled={waiting}
+                                value={acknowledgedDocuments}
+                                checked={acknowledgedDocuments}
+                                required
```

Comment:
If we're `waiting` then we are safe to `disable` the field. Also, we want the checkbox to show as `checked` for sure.